src/libraries/System.IO.Pipes.AccessControl/README.md

```diff
@@ -0,0 +1,14 @@
+# System.IO.Pipes.AccessControl
+This assembly no longer contains executable code, it contains the code for the reference assembly and the tests for the types formerly implemented.
+
+## Source
+* All types previously implemented on this library are now part of [System.IO.Pipes](../System.IO.Pipes/) library.
+
+* Some of the tests for types previously in this library are still in the [tests](tests/) subdirectory.
+
+## Contribution Bar
+- [x] We consider changes that move tests from the [tests](tests/) subdirectory into [System.IO.Pipes.Tests](../System.IO.Pipes/tests/) project.
+- [x] We encourage you to add new tests in [System.IO.Pipes.Tests](../System.IO.Pipes/tests/) project.
```

Comment:
Please let me know what you think of these paragraphs. It seems to me that the tests here can be moved to the assembly where the types now live. 


I was wondering. Is it possible to remove the System.IO.Pipes.AccessControl folder and just ship the assembly? Similar to how other assemblies ship today e.g: `System.IO.Compression.FileSystem.dll` which doesn't have a folder in src/libraries at all, that may help to avoid confusions in the future.

Reply:
I assume you're basing this on what I did for https://github.com/dotnet/runtime/blob/main/src/libraries/System.Security.Cryptography.Cng/README.md:

* We consider changes that move tests from the tests subdirectory into the unified System.Security.Cryptography.Tests project.
* We consider changes that add value to the tests, provided they cannot easily or effectively be added to the unified System.Security.Cryptography.Tests project.

In that case, there's a technical reason that the tests haven't been all moved over yet (the tests were designed as shared-source with one RSA (etc) type per tested assembly).  So it's saying "if there's a test that's good to have, but fits the old/current design, we're not going to say no".  Otherwise, the first bullet suggests we'd say no.

If this assembly is "yeah, we should move the tests...." then I'd leave out the second bullet, as (to me) it's implied by the first, as well as the fact that all the types have moved.

Reply:
> Is it possible to remove the System.IO.Pipes.AccessControl folder and just ship the assembly? Similar to how other assemblies ship today e.g: System.IO.Compression.FileSystem.dll

Beats me.  If there is, that'd be good to do for about half of System.Security.Cryptography.* :smile:

Reply:
Filed https://github.com/dotnet/runtime/issues/78978 for discussion.